diff --git a/src/agent/launcher.rs b/src/agent/launcher.rs
--- a/src/agent/launcher.rs
+++ b/src/agent/launcher.rs
@@ -1046,7 +1046,7 @@
                 ));
             };
 
-            if add_virtiofs3(ctx, root_tag.as_ptr(), root.as_ptr(), 0, false) < 0 {
+            if add_virtiofs3(ctx, root_tag.as_ptr(), root.as_ptr(), 0, true) < 0 {
                 krun_free_ctx(ctx);
                 return Err(Error::agent(
                     "set rootfs",
@@ -1071,7 +1071,7 @@
                 root_tag.as_ptr(),
                 root.as_ptr(),
                 rootfs_dax_window,
-                false,
+                true,
             ) < 0
             {
                 krun_free_ctx(ctx);
@@ -2048,7 +2048,7 @@
                         tag.as_ptr(),
                         host_path.as_ptr(),
                         super::virtiofs::packed_layers_dax_window(),
-                        false,
+                        true,
                         super::krun::KRUN_VIRTIOFS_FLAG_OVERRIDE_STAT,
                     )
                 } else {
@@ -2064,6 +2064,6 @@
                         tag.as_ptr(),
                         host_path.as_ptr(),
                         super::virtiofs::packed_layers_dax_window(),
-                        false,
+                        true,
                     )
                 };
